sdk/python/lib/test/langhost/stack_output/__main__.py

```diff
@@ -34,3 +39,5 @@ def __init__(self):
 pulumi.export("output", pulumi.Output.from_input(1))
 pulumi.export("class", TestClass())
 pulumi.export("recursive", recursive)
+pulumi.export("duplicate_output_0", my_duplicate_class.id)
```

Comment:
Do you even need DuplicateOutputClass? Couldn't you just return the 1 twice?

Reply:
but this is a primitive export, it's just 1. This would be more interesting returning the same array or dict instance twice.

Reply:
Ah yeah - I see your point.  I'll update this test to return the same class instance and assert against the serialized keys here.